config/routes.rb

```diff
@@ -1,3 +1,22 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: 'items#index'
+  get '/merchants', to: 'merchants#index'
+  get '/merchants/new', to: 'merchants#new'
+  post '/merchants', to: 'merchants#create'
+  get '/merchants/:id', to: 'merchants#show'
+  get '/merchants/:id/edit', to: 'merchants#edit'
+  patch '/merchants/:id', to: 'merchants#update'
+  delete '/merchants/:id', to: 'merchants#destroy'
+  get '/merchants/:merchant_id/items', to: 'items#index'
+  get '/merchants/:merchant_id/items/new', to: 'items#new'
+  get '/merchants?*all', to: 'merchants#index'
+  get '*/items?*all', to: 'items#index'
```

Comment:
you don't necessarily need these two routes above. It will still route to teh merchants index and the items index after you add queries to it.